packages/story-editor/src/components/panels/design/layer/layerPanel.js

```diff
@@ -57,8 +58,10 @@ function LayerPanel() {
   } = useInspector();
 
   // We want the max height to fill the space underneath the document/design tab bar.
-  // Since the document/design tab bar is 50px subtract that from the full height of the inspector.
-  const maxHeight = inspectorContentHeight - 50;
+  // Since the document/design tab bar has top and bottom margin in addition to a static
+  // height subtract sum from the full height of the inspector.
+  const tabHeight = 2 * TAB_VERTICAL_MARGIN + TAB_HEIGHT;
+  const maxHeight = inspectorContentHeight - tabHeight + 2;
```

Comment:
nit: could memoize this since these are all static (unless the user resizes the window). We don't save much by memoizing this though, it's only a couple calculations.

Reply:
I think memoizing in this case would end up being more expensive than the simple math 🧐

Reply:
sounds good!